keras_spiking/regularizers.py

```diff
@@ -16,27 +80,34 @@ class L1L2(tf.keras.regularizers.L1L2):
         Weight on L1 regularization penalty.
     l2 : float
         Weight on L2 regularization penalty.
-    target : float
-        Target output value (values will be penalized based on their distance from
-        this point).
+    target : float or tuple
+        The value that we want the regularized outputs to be driven towards. Can be a
+        float, in which case all outputs will be driven towards that value, or a tuple
+        specifying a range ``(min, max)``, in which case outputs outside that range
+        will be driven towards that range (but outputs within the range will not be
+        penalized).
     """
 
     def __init__(self, l1=0.0, l2=0.0, target=0, **kwargs):
```

Comment:
Should we have a warning if both `l1` and `l2` are 0? I could see users not changing these (since there are defaults), and being surprised when nothing happens.

Reply:
This is designed to mimic https://www.tensorflow.org/api_docs/python/tf/keras/regularizers/L1L2, where they both default to 0

Reply:
Oh nm you were just saying add a warning, not change the defaults. Yeah that sounds good :+1: